lib/kb_uploadmethods/Utils/ImportReadsUtil.py

```diff
@@ -14,87 +14,95 @@ def __init__(self, config):
         self.uploader_utils = UploaderUtil(config)
         self.sra_importer = ImportSRAUtil(config)
 
-    def import_reads_from_staging(self, params):
-        self._validate_import_reads_from_staging_params(params)
-
-        if params.get('import_type') == 'FASTQ/FASTA':
-            fastq_importer_params = params
-            fastq_importer_params['fwd_staging_file_name'] = params.get(
-                                                        'fastq_fwd_staging_file_name')
-            fastq_importer_params['rev_staging_file_name'] = params.get(
-                                                        'fastq_rev_staging_file_name')
+    def _run_fastq_importer(self, params):
+        fastq_importer_params = params
+        fastq_importer_params['fwd_staging_file_name'] = params.get(
+            'fastq_fwd_staging_file_name')
+        fastq_importer_params['rev_staging_file_name'] = params.get(
+            'fastq_rev_staging_file_name')
 
-            returnVal = self.uploader_utils.upload_fastq_file(fastq_importer_params)
+        return_val = self.uploader_utils.upload_fastq_file(fastq_importer_params)
 
-            uploaded_file = params.get('fastq_fwd_staging_file_name')
-            if params.get('fastq_rev_staging_file_name') is not None:
-                uploaded_file += '\n' + params.get('fastq_rev_staging_file_name')
-            fastq_importer_params['uploaded_files'] = [uploaded_file]
+        uploaded_file = params.get('fastq_fwd_staging_file_name')
+        if params.get('fastq_rev_staging_file_name') is not None:
```

Comment:
minor but would it make sense to just do one call for `params.get('fastq_rev_staging_file_name')`? e.g. 
 ```
   name = params.get('fastq_rev_staging_file_name')
   if name is not None: 
     updated_file += '\n' + name 
 ```